server/api/models.py

```diff
@@ -79,7 +79,10 @@ class Respuesta(models.Model):
     auditoria = models.ForeignKey(Auditoria, on_delete=models.CASCADE)
     respuesta = models.CharField(max_length=128, null=True, blank=True)
     notas = models.TextField(max_length=256, null=True, blank=True)
-    audio = models.FileField(upload_to='audios_de_respuesta/', null=True, blank=True)
+    if settings.USE_S3:
+        audio = models.FileField(storage=PublicMediaStorage(), null=True, blank=True)
+    else:
+        audio = models.FileField(upload_to='audios_de_respuesta/', null=True, blank=True)
```

Comment:
Tengo entendido que no esta bueno tener lógica dentro de un modelo. Lo que se hace generalmente si queres guardar un archivo dependiendo de ciertas condiciones, es pasarle al FileField una función, en vez de un path.

Por poner un ejemplo:
```
def user_directory_path(instance, filename):
    # file will be uploaded to MEDIA_ROOT/user_<id>/<filename>
    return 'user_{0}/{1}'.format(instance.user.id, filename)

class MyModel(models.Model):
    upload = models.FileField(upload_to=user_directory_path)
```
Te recomiendo que le pegues una leidita a la sección File Field para entenderlo bien, literal son solo 2 minutos:
https://docs.djangoproject.com/en/3.2/ref/models/fields/#filefield

Ahora, por otro lado veo que además utilizas un parámetro llamado storage, la verdad nunca lo había utilizado, pero no me suena que sea el mejor camino.
A priori, para guardar los datos, hubiera pensado crear una función como te mostré anteriormente, y hacer la lógica que hiciste ahi dentro. Además, por lo que estuve mirando, no se si es una buena práctica pasarle el storage como argumento. Me parece que eventualmente nos va a dar mas dolores de cabeza. Capaz me equivoco, pero en algun post de stack overflow leí eso. Por ejemplo aca: https://stackoverflow.com/questions/59437637/django-use-private-s3-storage-only-in-production-environment

Capaz podemos juntarnos y ver que hace bien ese parámetro para ver como sacarlo de ahí.
@JopeAlgorta @jopealgorta24 Vos alguna vez utilizaste el "storage"?